src/domain/expense/service.py

```
@@ -117,15 +123,19 @@ async def get_by_id(self, object_id: Any, user_id: UUID) -> Expense:
+        page: int = 1,
+        limit: int = 20,
+        date_filter: datetime.date | None = None,
+        start_date: datetime.date | None = None,
+        end_date: datetime.date | None = None,
         min_value: Decimal | None = None,
         max_value: Decimal | None = None,
-    ) -> list[Expense]:
-        statement = select(Expense).where(Expense.user_id == user_id)
+    ) -> PaginatedResponseSchema:
+        if limit < 1:
+            raise
```

Ruff (0.15.9)

[error] Bare `raise` statement is not inside an exception handler

(PLE0704)